ArduCopter/mode_guided.cpp

```diff
@@ -408,8 +446,8 @@ void ModeGuided::takeoff_run()
 #endif
 
         // switch to position control mode but maintain current target
-        const Vector3f target = wp_nav->get_wp_destination();
-        set_destination(target, false, 0, false, 0, false, wp_nav->origin_and_destination_are_terrain_alt());
+        const Vector3f target = pos_control->get_pos_target_cm();
```

Comment:
This removes the ability to use the rangefinder as part of the takeoff in Guided mode which undoes the work of PR https://github.com/ArduPilot/ardupilot/pull/13768.  I think we need to think about this and how to maintain this existing feature.